engine/installation/cloud/overview.md

```diff
@@ -4,15 +4,9 @@ aliases:
 description: Installation instructions for Docker on cloud.
 keywords:
 - cloud, docker, machine, documentation,  installation
-menu:
-  main:
-    parent: install_cloud
-    weight: -3
 title: Choose how to install
```

Comment:
I hate this title. Maybe "Docker installation choices"? or "Choose your installation method"?